test/index.test-d.ts

```diff
@@ -49,6 +49,17 @@ await packager({
   },
 });
 
+await packager({
+  dir: '.',
+  name: 'myapplication',
+  platform: 'darwin',
+  arch: 'universal',
+  electronVersion: '0.34.0',
```

Comment:
This isn't a great example since there's no arm binary for that package, but 🤷‍♂️